Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM debian:buster-slim
```

Comment:
Instead of using `debian:buster-slim` as base you could use `python:3.10.1-(slim-)buster` for hassle-free python version management. But using the default version of `buster-slim` is also ok